pkgs/os-specific/linux/busybox/default.nix

```
@@ -64,6 +67,10 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./busybox-in-store.patch
+    (fetchpatch {
+      url = "https://git.busybox.net/busybox/patch/?id=f25d254dfd4243698c31a4f3153d4ac72aa9e9bd";
```

```suggestion
      name = "CVE-2021-28831.patch";
      url = "https://git.busybox.net/busybox/patch/?id=f25d254dfd4243698c31a4f3153d4ac72aa9e9bd";
```

Giving the patch a name makes tracking fixed CVEs easier.